app/assets/stylesheets/framework_and_overrides.css.scss

```diff
@@ -102,3 +102,52 @@ img {
   height: 25vh;
   background-color: rgba(255,255,255,0.5);
 }
+
+.apartment-info {
+  background-color: #c5499b;
+  box-sizing: border-box;
+  margin: 5px 0;
+  display: block;
+  border-radius: 4px;
+  color: #fff;
+  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  padding: 5px 15px;
+  text-align: center;
+  width: auto;
+  font-size: 24px;
+}
+
+.apartment-metric {
+  background-color: #49c5b1;
```

Comment:
Color literals like `#49c5b1` should only be used in variable declarations; they should be referred to via variable everywhere else.